testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -167,7 +167,27 @@ def test_write_frame_none(self, u, tmpdir, extension):
             u.atoms.positions[None, ...], new_positions, decimal=2
         )
 
+    @pytest.mark.parametrize('extension', ('xtc', 'dcd', 'pdb', 'xyz'))
+    def test_compressed_write_frame_none(self, u, tmpdir, extension):
+        for ext in ('.gz', '.bz2'):
+            destination = str(tmpdir / 'test.' + extension + ext)
+            u.atoms.write(destination, frames=None)
+            u_new = mda.Universe(destination)
+            new_positions = np.stack([ts.positions for ts in u_new.trajectory])
+            assert_array_almost_equal(
+                u.atoms.positions[None, ...], new_positions, decimal=2
+        )
+
     def test_write_frames_all(self, u, tmpdir):
+        for ext in ('.gz', '.bz2'):
+            destination = str(tmpdir / 'test.dcd') + ext
+            u.atoms.write(destination, frames='all')
+            u_new = mda.Universe(destination)
+            ref_positions = np.stack([ts.positions for ts in u.trajectory])
+            new_positions = np.stack([ts.positions for ts in u_new.trajectory])
+            assert_array_almost_equal(new_positions, ref_positions)
+
+    def test_compressed_rite_frames_all(self, u, tmpdir):
```

Comment:
"rite"?? Do you mean "right"? or "write"?

Reply:
So sorry. Did it in a hurry.